pkg/controller/acicontainersoperator.go

```diff
@@ -138,6 +181,22 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 
 	c.Logger.Info("Controller.Run: initiating")
 
+	log.Info("Checking if Custom Resource already present")
+	err := c.GetAciContainersOperatorCR()
+	if err != nil {
+		log.Info("Not Present ..Creating Custom Resource")
```

Comment:
Can you add the name of the custom resource (we know there is only one, but it will still make it more readable in the logs).